.github/actions/conditional/conditions

```diff
@@ -0,0 +1,14 @@
+# File patterns used to decide what workflows/jobs to execute for a given PR
+#
+# To test a pattern run '.github/actions/conditional/conditional.sh <remote name> <branch>'
+# Alternatively, run 'git diff <remote name>/<branch> <pattern>' (from the root directory)
+#
+# Note: The file needs to end with an empty line, otherwise last pattern is not picked up
+
+ci=*/pom.xml */src/main/ */src/test/
+
+operator=*/pom.xml */src/main/ */src/test/
+
+codeql-java=*.java
+codeql-themes=themes/
+codeql-js_adapter=adapters/oidc/js/
```

Comment:
The note above mentions that the file should end with an empty line. Is it missing?